frontend/src/components/cronjob/Details.tsx

```diff
@@ -137,7 +137,7 @@ export default function CronJobDetails() {
   const { t, i18n } = useTranslation('glossary');
 
   const [jobs, jobsError] = Job.useList({ namespace });
-  const [cronJob, setCronJob] = useState<CronJob | null>(null);
+  const [cronJob] = CronJob.useGet(name, namespace);
```

Comment:
Supposedly this was done that way because the DetailsGrid already fetches the CronJob. I know that with react-query, we are not re-fetching things multiple times, but I wonder if this was needed in this commit.

Reply:
yeah it's not required, reverted